src/bindings/python/pyproject.toml

```diff
@@ -13,7 +13,6 @@ exclude = [
     "src/openvino/_op_base.py",
     "src/openvino/utils/node_factory.py",
     "src/openvino/frontend/tensorflow",
-    "src/openvino/frontend/pytorch",
```

Comment:
Why this change?

Reply:
there is a list of files to exclude from pyright checks, so as we want to analyze also python files in pytorch folder, this path should be removed from the list

Reply:
That list of paths/files is just a list of skips for `pyright` code analysis. The line should actually be deleted, now PT FE is actually being tested on CI.